saul-core/src/main/scala/edu/illinois/cs/cogcomp/saul/classifier/infer/InitSparseNetwork.scala

```diff
@@ -0,0 +1,43 @@
+package edu.illinois.cs.cogcomp.saul.classifier.infer
+
+import edu.illinois.cs.cogcomp.lbjava.learn.{ LinearThresholdUnit, SparseNetworkLearner }
+import edu.illinois.cs.cogcomp.saul.classifier.ConstrainedClassifier
+import edu.illinois.cs.cogcomp.saul.datamodel.node.Node
+
+/** Created by Parisa on 9/18/16.
+  */
+object InitSparseNetwork {
+  def apply[HEAD <: AnyRef](node: Node[HEAD], cf: ConstrainedClassifier[_, HEAD]) = {
+    val allHeads = node.getTrainingInstances
+    //this means we are not reading any model into the SparseNetworks but we forget all the models and go over the data to build the right size for the lexicon and the right number of the ltu s
```

Comment:
minor: split comment to multiple lines? We should try not to exceed 100 characters column.